src/server/rest/v1/service/CarService.ts

```diff
@@ -582,7 +582,7 @@ export default class CarService {
         UtilsService.assertObjectExists(action, foundUser, `User ID '${userToCheck.user.id}' does not exist`,
           MODULE_NAME, 'handleAssignCarUsers', loggedUser);
         // Auth
-        if (!(await Authorizations.canReadUser(loggedUser)).authorized) {
+        if (!(await Authorizations.canReadUser(loggedUser, { UserID: loggedUser.id })).authorized) {
```

Comment:
Just FI, this will not be the final solution

Reply:
Yes, I will create an issue.